### DISCUSSION TITLE
It's sad to see how far Status has fallen.

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#21839] [General]
It's sad to see how far Status has fallen.

It used to be a project for the people to resist censorship and people in legitimately oppressive regimes to communicate freely and learn about the things that were banned where they lived.

The communities feature was a complete failure, the most most popular community was ran by a criminal organization, right behind the one from the company that makes the app, which was only "popular" because it came pre-loaded in the app. To put that in perspective, the Status community hit 800 members in about 3 months. SimpleX Chat (a better messaging app which also happens to be much newer with a team of 6 total) gains about 1000 members in its official community every few months (1-2 usually), and that one isn't even preloaded or listed anywhere in the app, you have to search online to find it.

There are over 10,000 active public SimpleX groups with over 100 members in them. There's maybe a dozen Status communities total despite the marketing hype.

Despite pouring millions into a redesign (which most users hated), the app is still slow, buggy, resource intensive, and basic features aren't there such as file sharing and VoIP. On the topic of features, the 2.0 update took away voice messages and a few other messaging features, so its basically just a worse uTox with crypto bloat now... except its not, because they added telemetry!

Telemetry in a privacy product? GTFO! If you enable analytics in the app, almost everything you do sends events to centralized servers. Every screen you open, every time you log in, every keycard you use, and data about messages you receive (but no contents [yet]). This is actually worse than Tox because it uses dark patterns to try to get the user to click agree on the telemetry prompt. Grrrr

Features silently cancelled since the initial 2.0 announcement:
- Being a super app
- Unified tabs for messaging, browser, communities, and wallet
- Mini-apps
- Voice messages
- Profile customization
- Massively scalable communities
- Telegram bridging
- Social importing
- The entire browser tab
- Multi-message chaining with Waku
- The ability to search messages or really anything (come on, its a SQL database)
- Community discovery (sure theres two communities in there, but you MUST hold their shitcoin to even have the opportunity)
- High quality media uploads
- Link to specific message in channel
- Message bubbles (far superior to discord-style)

Now its just an Ethereum-only crypto wallet that wants you to buy with KYC and has a barebones messaging feature that's a little better than Discord on the privacy front with a somewhat decent UI, as well as pushing telemetry on users.

Not what I signed up for. [SimpleX Chat](https://simplex.chat) is my new home, but thanks for the interesting adventure and opportunity to write code and dive deep into the tech behind what makes cryptocurrency actually work.

**Comment by du82** [comment]
@86doteth thoughts?

**Comment by 86doteth** [comment]
glad to see youve marginally improved from shilling matrix to shilling simplex. one day you might also appreciate that building a messenger where the set of private relayers doesnt just consist of either professionals or die hard-altruists running server infra but actually most _users_ of the app is the actual order of magnitude improvement. absolutely no one is competing with status on this, and i dont blame them because its hard af. yet it has to be done and status is doing it.

centralized telemetry has been completely [open-source](https://github.com/status-im/telemetry) and opt-in, mostly served its much-needed purpose of waku reliability debugging (thanks to which [message reliability](https://forum.vac.dev/t/ift-research-call-december-4th-2024-reliability-over-waku/404) is looking to finally get solved in the [coming months](https://github.com/waku-org/pm/milestone/29) after many years) and is about to be [sunset for local telemetry](https://forum.vac.dev/t/the-future-of-status-telemetry/405).

you polemically declare that status 'used to be' for censorship resistance and learning about banned stuff and then literally state in the next sentence that one of the popular communities was a criminal one ROFL. also status inu is bigger than the one you mentioned and there might be many potentially larger communities out there without an owner token for which we dont know the link. the one you mentioned is only known to us because someone posted a link to it in the status community.

> The communities feature was a complete failure,

it was released in beta just a few months ago, still is, is [actively developed](https://github.com/status-im/status-desktop/issues?q=is%3Aissue%20label%3Agoal%3Acore%3AP1%3Acommunity_beta) with fewer and fewer bugs each release and is waiting for a few message reliability and performance fixes to hit before it can be actively marketed and handle large user influx

> There's maybe a dozen Status communities total despite the marketing hype.

which is why youre wrong here as well. they literally hid the 'create community' button on desktop because the feature is not quite ready for mass use yet. basically only die-hards are using the feature right now which is a good call. idk what you call marketing hype, they tweeted a few dozen times about it over the last year after almost radio-silence for two years because they were heads-down building. the only highlight is maybe the relaunch of the website, so it matches the product suite now. wow such marketing.

> because it came pre-loaded in the app.

its not pre-loaded, it passed an snt vote to be listed in the [actually decentralized community directory](https://status.app/help/communities/about-voting-to-change-the-community-visibility), just like status inu did and just like anyone else can. [as opposed to simplex ](https://simplex.chat/docs/directory.html) where only the company decides who gets listed LMAO

> Despite pouring millions into a redesign (which most users hated), the app is still slow, buggy, resource intensive, and basic features aren't there such as file sharing and VoIP. On the topic of features, the 2.0 update took away voice messages and a few other messaging features

idk where you got 'most users' from. only positive comments ive seen have been about the new ui. most complaints ive seen in the app seem more about missing features. the missing features issue is nothing else but a matter of prioritization: the features that we have now and in the direct pipeline need to become more performant and bug free before we stack more advanced features on top.

>Features silently cancelled since the initial 2.0 announcement:
>
>- Being a super app

matter of prioritization and matching expectation to current state of app. feature dev timelines were estimated too optimistically in the past but as explained above status is competing only with itself in a league of its own due to the underlying privacy-superior waku network that also makes new features harder to develop. dropping superapp terminology is fine for now to manage expectations of users until more features are built out.

>- Mini-apps

this is the whole point of the upcoming status network l2 so def not silently dropped

>- Massively scalable communities

this is literally worked on in multiple waku [milestones](https://roadmap.logos.co/waku/waku-milestones) together with status. smaller community descriptions, de-mls etc.

>- The entire browser tab

this saddens me as well but for now the app has walletconnect and the upcoming status [browserconnect](https://github.com/status-im/status-web/tree/main/apps/connector) plugin to interact with any dapp. i think its indeed a better call to actually make the current features actually work reliably and performant before spending more precious resources on extra ui features

the rest of your list would all be indeed great to have, but the amount of work it takes to make all these work on gossipsub compared to a few beefy 'relaying' servers is just a whole other level of difficulty such that it explains them needing to take the time for this. im sure they will eventually get to all of these.

>Now its just an Ethereum-only crypto wallet

youre completely, probably wilfully, missing how big a part this is of the value offering status brings. the wallet features take about 50% of resources so that is another thing you have to weigh in terms of development time against any messenger you want to compare against (simplex and none of the other big ones have built-in wallets at all). they even developed a [fully open-source hardware wallet](https://keycard.tech/) in-house that is about to work on both desktop and mobile, that doesnt only protect your funds but even the keys to your community ownership, so no more hacked discord servers. as if that wasnt enough they will soon offer a [fully airgapped upgrade of keycard](https://x.com/ethstatus/status/1806001469973278770?s=46) that will allow you to use status as the place where you can keep most of your holdings, which alone would make it competitive with metamask and ledger. finally the long term vision has always been to have [nimbus](https://nimbus.team/) as a built-in validator/full node and ultra-light ethereum client on respectively desktop and mobile, which admittedly is still more longer timeframe. in any case it will play a [central](https://discuss.status.app/t/nip-3-launch-of-status-layer-2-rollup-chain/4143/3?u=86.eth) role in the soon to launch status l2.

>wants you to buy with KYC

... as one of many options. its a fully fledged ethereum wallet that you can fund however you like. it even has a built-in dex. in the end all blockchain wallets, even monero ones, need fiat on-ramps which are nowadays pretty much all kyc'ed.

>[SimpleX Chat](https://simplex.chat/) is my new home

i think youve announced your departure to simplex chat a bunch of times already in recent months. its not an airport etc

**Comment by Nika250** [comment]
This is important to me too, thank you.
